lib/copy_peste/command/run_analysis_module.rb

```diff
@@ -19,14 +19,14 @@ def run
 
         ar = AnalyseResult.new
         begin
-          @core_state.analysisModule.run ar
+          ret = @core_state.analysisModule.run ar
         rescue Exception => e
           traceback = ""
           traceback = "(Traceback) #{e}" if !e.message.empty?
           @graph_com.cmd_return(@cmd,
             "Analysis module aborted.\n#{traceback}", true)
         end
-        ar.save!
+        ar.save! if ret == true
```

Comment:
Don't check if value is `true` but if it is truthy (a `true` evaluated value). You don't have to rely on `@core_state.analysisModule.run` method return.